app/graphql/types/resource.rb

```diff
@@ -19,7 +19,7 @@ def resolve_type(object, _context)
   end
 
   def members
-    @members ||= Wayfinder.for(object).members
+    @members ||= Wayfinder.for(object).members_with_parents
```

Comment:
Can you add a comment here explaining that this retrieves the members with the parent pre-cached?